scripts/install-chef-suse.sh

```diff
@@ -278,6 +278,15 @@ post_fail_handler ()
     rm -f $crowbar_install_dir/crowbar_installing
 }
 
+function reset_crowbar()
+{
+    rm -f $crowbar_install_dir/crowbar-install-failed
+    rm -f $installation_steps
+    pushd /opt/dell/crowbar_framework > /dev/null
+    bin/rake db:cleanup
```

Comment:
That should not matter as it's not creating files, it's just migrating down and up an existing database.